source/_includes/hardware-platforms.html

```diff
@@ -1,112 +1,164 @@
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
 {% endcomment %}
 
 <table class="centered-table">
-  <thead>
-    <tr>
-      <th>Feature</th>
-      <th>Classic,<br />Steel</th>
-      <th nowrap="nowrap">Time,<br />Time Steel</th>
-      <th>Time Round</th>
-      <th>Pebble 2</th>
-      <th>Time 2</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>Platform</td>
-      <td>Aplite</td>
-      <td>Basalt</td>
-      <td>Chalk</td>
-      <td>Diorite</td>
-      <td>Emery</td>
-    </tr>
-    <tr>
-      <td>CPU</td>
-      <td nowrap="nowrap">Cortex-M3<br />64 MHz</td>
-      <td colspan="3">Cortex-M4<br />100 MHz</td>
-      <td>Cortex-M7<br />144 MHz</td>
-    </tr>
-    <tr>
-      <td>Max. resource size</td>
-      <td>96k</td>
-      <td colspan="3">256k</td>
-      <td>512k</td>
-    </tr>
-    <tr>
-      <td>Max. app size (code + heap)</td>
-      <td>24k</td>
-      <td colspan="3">64k</td>
-      <td>128k</td>
-    </tr>
-    <tr>
-      <td>Display shape</td>
-      <td colspan="2">Rectangle</td>
-      <td>Round</td>
-      <td colspan="2">Rectangle</td>
-    </tr>
-    <tr>
-      <td>Display resolution</td>
-      <td colspan="2">144 x 168</td>
-      <td nowrap="nowrap">180 x 180</td>
-      <td nowrap="nowrap">144 x 168</td>
-      <td nowrap="nowrap">200 x 228</td>
-    </tr>
-    <tr>
-      <td>Display PPI</td>
-      <td>175</td>
-      <td colspan="2">182</td>
-      <td>175</td>
-      <td>202</td>
-    </tr>
-    <tr>
-      <td>Supported colors</td>
-      <td>2</td>
-      <td colspan="2">64</td>
-      <td>2</td>
-      <td>64</td>
-    </tr>
-    <tr>
-      <td>Heart Rate Monitor</td>
-      <td>No</td>
-      <td colspan="2">Yes<br /><em>(with smartstrap)</em></td>
-      <td>Yes<br /><em>(except SE model)</em></td>
-      <td>Yes</td>
-    </tr>
-    <tr>
-      <td>Microphone</td>
-      <td>No</td>
-      <td colspan="4">Yes</td>
-    </tr>
-    <tr>
-      <td>Sensors</td>
-      <td colspan="3">Accelerometer, Compass</td>
-      <td>Accelerometer</td>
-      <td>Accelerometer, Compass</td>
-    </tr>
-    <tr>
-      <td>Buttons</td>
-      <td colspan="5">4</td>
-    </tr>
-    <tr>
-      <td>Charging port</td>
-      <td>Power only</td>
-      <td colspan="2">Smart accessory port</td>
-      <td>Smart accessory port <em>(data only)</em></td>
-      <td>Smart accessory port</td>
-    </tr>
-  </tbody>
+    <thead>
+        <tr>
+            <td>Manufacturer</td>
+            <th colspan="5">Pebble</th>
+            <th colspan="2">Core Devices</th>
+        </tr>
```

Comment:
This header doesn't render very well - can you clean it up a bit?
<img width="776" height="118" alt="image" src="https://github.com/user-attachments/assets/0d94d594-36d1-40e8-8a1c-70bc2a00c0fb" />

Reply:
Ok, I'll see what I can do. I'll try moving manufacturer underneath the device names and disable it from being a heading.

Reply:
I think it should stay where it is - but there should be a border between the two rows, and the Manufacturer/Model part should be the same colour as the other cells.

Reply:
I missed your reply above. I just redid the header.

Reply:
Definitely still not looking right - the table appears to have lost the header styling completely.
<img width="659" height="379" alt="image" src="https://github.com/user-attachments/assets/78628822-20e8-4199-ae7d-08193bee1150" />

Reply:
Oh i removed it. I misunderstood what you were asking. I'll add it back.

Reply:
I have made the changes. Hopefully this is better.